runner/Dockerfile

```diff
@@ -113,6 +113,10 @@ RUN mkdir /opt/hostedtoolcache \
 # We place the scripts in `/usr/bin` so that users who extend this image can
 # override them with scripts of the same name placed in `/usr/local/bin`.
 COPY entrypoint.sh logger.bash /usr/bin/
+RUN chmod 755 entrypoint.sh
+
+COPY job_start.sh job_end.sh update_status.sh /
+RUN chmod 755 job_start.sh job_end.sh update_status.sh
```

Comment:
Why would we not install the scripts into `/usr/bin` and thus allow resolution from `PATH` and users to override it with their own scripts in `/usr/local/bin`?

I see no reason for the `.sh` suffix for the filenames, any reason for it? Personally I dislike underscores (`_`) in filenames because it is an old Unix tradition to use hyphens (`-`), a tradition that came into existence before most of us where even born. That said, the scripts are so simple, the could be combined into a single one. Unless we want to give users the ability to override them, in which case placing them in `/usr/bin` is again the more logical choice. Or, what @mumoshu proposed, a dedicated directory where the should be stored too, in this case I would propose the following that follows Unix conventions once more:

```
/etc/arc/runner/job
+-- started
+-- started.d/
|   +-- user-provided-script
|   +-- another-user-provided-script
|   +-- ...
+-- stopped
+-- stopped.d/
    +-- user-provided-script
    +-- another-user-provided-script
    +-- ...
```